libs/designsystem/src/lib/components/toggle-button/toggle-button.module.ts

```diff
@@ -2,9 +2,8 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
 import { ToggleButtonComponent } from './toggle-button.component';
-import { ToggleButtonThemeColorDirective } from './directives/toggle-button-color.directive';
 
-const DECLARATIONS = [ToggleButtonComponent, ToggleButtonThemeColorDirective];
+const DECLARATIONS = [ToggleButtonComponent];
 
 @NgModule({
   declarations: [...DECLARATIONS],
```

Comment:
Does it make sense to have a module for 1 component? We don't do that elsewhere...

Reply:
Does it make sense to create a class that only contains one public method, same thing...

I believe every ui component should have it's own module.
Check Angular Material:
https://github.com/angular/components/blob/master/src/material/button/button-module.ts

They create a module for each component, so everything necessary for that UI component is encapsulated in the module.